conf.d/downloads

```diff
@@ -6,7 +6,7 @@ dl() {
 }
 
 SRC=/usr/local/src
-VERSION="6.11.6"
+VERSION="7.2.2"
```

Comment:
It might be nice to leverage `gh_releases` to get the latest release? Thoughts? Or should we just leave it for now and catch that up later at some point?

Reply:
We cannot reliably use gh_releases yet as their tag names do not include information for determining if they are stable or not. e.g:

v7.2.1 is beta, v7.2.2 is stable but their tags are `7.2.1` and `7.2.2` respectively. Eventually we should update gh_releases to properly handle github releases that don't necessarily map to tags. But for now we'll have to continue manually.

Reply:
Ah ok. That's a pity we can't automate it, but your rationale certainly makes sense! :smile: :+1: